tests/tests.cpp

```diff
@@ -326,6 +353,16 @@ TEST_F(EngineBasicTest, TestBasicHashOperations) {
       AssignData(v1, v1_len);
       AssignData(v2, v2_len);
 
+      if (id == 0) {
+        std::string key{""};
+        std::string value{"Empty key but with value"};
+        std::string sink{};
+        ASSERT_EQ(engine->Set(key, value), Status::Ok);
+        ASSERT_EQ(engine->Get(key, &sink), Status::Ok);
+        ASSERT_EQ(value, sink);
+        ASSERT_EQ(engine->Delete(key), Status::Ok);
```

Comment:
check if Get returns not found after Delete

Reply:
add ASSERT_EQ for Get after Delete.